src/components/Album.jsx

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable react/prop-types */
+import { AlbumName } from "./AlbumName";
+import { ArtistName } from "./ArtistName";
+import { CoverImage } from "./CoverImage";
+import "./Album.css"; 
+export const Album = (props) => {
+  
+  let albumName = props.album.name
+  let artistName = props.album.artists[0].name
```

Comment:
I had my code like this aswell to start with, but apparently some albums have 2 artists, and with this command we only select the first one. For example the album with Rihanna.

You could try `let artist = props.album.artists` and then do a map of that array down in the return-statement